### PR TITLE
add ffca model, and replace the metadata export to fit the json format

### PR DESCRIPTION
**Pull request by kinkarHan**
1. add ffca model
2. replace the metadata export to fit the json format

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
A major update introduces new model configurations and improvements to the YOLOv5 architecture, enhancing performance and flexibility.

### 📊 Key Changes
- **New Model Configurations:** Added `ffca_yolov5-seg.yaml`, `ffca_yolov5.yaml`, `l_ffca_yolov5.yaml` files.
- **New Modules:** Introduced several new neural network modules such as `FEM`, `SCAM`, `C3_Faster`, `FFM_Concat2`, and `FFM_Concat3`.
- **ONNX Export Enhancement:** Improved the metadata export process to use JSON format.

### 🎯 Purpose & Impact
- **Improved Performance:** Adding new configurable models and advanced modules enhances model accuracy and training efficiency.
- **Flexibility:** The new configurations and modules offer better flexibility for different tasks and datasets.
- **Enhanced Interoperability:** Using JSON for metadata in ONNX export improves compatibility and simplifies integration with other tools.

### CODE REVIEW

**Comment by glenn-jocher** [comment]
Thank you for your contribution! Your updates to the YOLOv5 architecture are impressive and will undoubtedly enhance the model's performance and flexibility. Here are some key points:

1. **New Model Configurations:** The addition of `ffca_yolov5-seg.yaml`, `ffca_yolov5.yaml`, and `l_ffca_yolov5.yaml` files will provide users with more options tailored to their specific needs.
2. **New Modules:** Introducing modules like `FEM`, `SCAM`, `C3_Faster`, `FFM_Concat2`, and `FFM_Concat3` is a significant step forward. These modules are likely to improve the model's accuracy and efficiency.
3. **ONNX Export Enhancement:** Switching the metadata export to JSON format is a smart move. It enhances interoperability and makes it easier to integrate with other tools and platforms.

For those looking to leverage these new features, here’s a quick example of how to load and train one of the new configurations:

```python
from ultralytics import YOLO

# Load the new FFCA YOLOv5 model
model = YOLO("ffca_yolov5.yaml")

# Train the model on a custom dataset
results = model.train(data="custom_dataset.yaml", epochs=100, imgsz=640)
```

If you encounter any issues or have further suggestions, please provide a reproducible example to help us diagnose and address the problem effectively. You can find more information on creating a minimum reproducible example [here](https://docs.ultralytics.com/help/minimum_reproducible_example).

Thank you again for your valuable contribution! 🚀
